remix.init/index.ts

```diff
@@ -102,30 +61,30 @@ const main = async ({ isTypeScript, packageManager, rootDirectory }) => {
     ephemeralDestroyWorkflowPath,
     rootPath,
     rootTestPath,
-    e2ePath
+    e2ePath,
```

Comment:
Could probably add the git-init here too to replace the example url with the user's url